src/Resources/views/http_message.html.twig

```diff
@@ -33,4 +33,4 @@
     </tbody>
 </table>
 
-<div class='httplug-http-body httplug-hidden'>{{ content|nl2br ?: '(This message has no captured body)' }}</div>
+<div class='httplug-http-body httplug-hidden'>{{ content|httplug_markup_body }}</div>
```

Comment:
should we keep the alternate text for when there is no body around?

Reply:
If there is no body, vardumper will show this, which is also representative enough, isn't it?
![image](https://user-images.githubusercontent.com/496233/131211768-556cd019-1942-4f8a-bfea-f21baa39af65.png)

But in case you think it's better to have old placeholder back let me know.

Reply:
i think depending on configuration, we do not record the body. it would make sense to not lead people to think that there was an empty body when actually the body was just not recorded. but i notice that we currently don't distinguish between "we recorded that there was an empty body" and "we did not record the body". i would prefer to bring back the message we had as that did not claim its one or the other. we could further improve things to distinguish between empty body and not recorded body, but thats out of scope for this pull request.

Reply:
Ok, done!